README.md

```diff
@@ -20,22 +20,28 @@ python -m venv env
 source env/bin/activate
 ```
 3. `python -m pip install poetry`
-4. `poetry install mozmlops`
+4. `poetry install`
+
+## Running tests
+
+Run `pytest` from the top-level directory to run the unit tests.
```

Comment:
optional: do you think it's worth calling out that you need to be logged to GCP?

Reply:
It's a good question; the error message if you're not logged in is pretty specific and helpful. In my perfect world, complete startup scripts and specific error messages would replace all READMEs, but we do not live in that world (yet....<insert evil laugh here>)

However right now we do assume a contributor has gcloud set up already, which...though true with our current team members, probably worth calling out in the README. I'll add it!